### PR TITLE
Type annotations

### PR DESCRIPTION
**Pull request by djesic**

### CODE REVIEW

File: wolk_gateway_module/json_firmware_update_protocol.py [hunk 1]
```diff
@@ -12,19 +12,16 @@
...
-    FirmwareUpdateProtocol,
-)
 from wolk_gateway_module.model.firmware_update_status import (
     FirmwareUpdateStatus,
-    FirmwareUpdateState,
-    FirmwareUpdateErrorCode,
 )
 from wolk_gateway_module.model.message import Message
+from wolk_gateway_module.protocol.firmware_update_protocol import (
+    FirmwareUpdateProtocol,
```
Comment (srdjastankovic) [inline wolk_gateway_module/json_firmware_update_protocol.py:23]: the same as above, standalone

File: wolk_gateway_module/json_status_protocol.py [hunk 2]
```diff
@@ -80,14 +82,15 @@ def is_device_status_request_message(self, message: Message) -> bool:
         return is_device_status_request
 
     def make_device_status_response_message(
-        self, device_key: str, device_status: DeviceStatus
+        self, device_status: DeviceStatus, device_key: str
```
Comment (srdjastankovic) [inline wolk_gateway_module/json_status_protocol.py:85]: Why DeviceStatus notation differ from others(word_word)?

File: wolk_gateway_module/interface/configuration_handler.py [hunk 3]
```diff
@@ -12,7 +12,9 @@
 #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
-from typing import Dict, Union, Tuple
+from typing import Dict
+from typing import Tuple
+from typing import Union
```
[the remark this answers is not in the document]
Comment (djesic) [reply wolk_gateway_module/interface/configuration_handler.py:17]: Reorder imports hook sorts them like this.

File: wolk_gateway_module/json_data_protocol.py [hunk 4]
```diff
@@ -12,20 +12,18 @@
...
-)
-from wolk_gateway_module.model.actuator_state import ActuatorState
+from wolk_gateway_module.model.actuator_command import ActuatorCommand
+from wolk_gateway_module.model.actuator_command import ActuatorCommandType
 from wolk_gateway_module.model.actuator_status import ActuatorStatus
 from wolk_gateway_module.model.alarm import Alarm
 from wolk_gateway_module.model.configuration_command import (
     ConfigurationCommand,
+)
+from wolk_gateway_module.model.configuration_command import (
```
[the remark this answers is not in the document]
Comment (djesic) [reply wolk_gateway_module/json_data_protocol.py:26]: Line length is over 79, so black formats it into brackets

File: wolk_gateway_module/json_data_protocol.py [hunk 5]
```diff
@@ -340,11 +349,7 @@ def make_sensor_readings_message(
         :returns: message
         :rtype: Message
         """
-        topic = (
-            self.SENSOR_READING
-            + self.DEVICE_PATH_PREFIX
-            + device_key
-        )
+        topic = self.SENSOR_READING + self.DEVICE_PATH_PREFIX + device_key
```
[the remark this answers is not in the document]
Comment (djesic) [reply wolk_gateway_module/json_data_protocol.py:352]: Line length formatting again

File: wolk_gateway_module/json_firmware_update_protocol.py (hunk 1 above)
Comment (djesic) [reply wolk_gateway_module/json_firmware_update_protocol.py:23]: Line length

File: wolk_gateway_module/json_status_protocol.py (hunk 2 above)
Comment (djesic) [reply wolk_gateway_module/json_status_protocol.py:85]: Class naming

**Review by srdjastankovic — APPROVED** [review]

**Pull request merged**